.github/workflows/wheel_linux_aarch64.yml

```diff
@@ -159,6 +155,8 @@ jobs:
 
           CIBW_BEFORE_TEST: |
             python -m pip install -r requirements-tests.txt
+            # FIXME: install PennyLane/latest to fix pennylane.devices.capabilities requirements
```

Comment:
Will we remove these two lines after the next release?

Reply:
Yes!
```suggestion
            # FIXME(v0.40.0): install PennyLane/latest to fix pennylane.devices.capabilities requirements
```